aldryn_events/tests/frontend/integration/specs/spec.add.delete.event.js

```diff
@@ -0,0 +1,38 @@
+/*!
+ * @author:    Divio AG
+ * @copyright: http://www.divio.ch
+ */
+
+'use strict';
+/* global describe, it, browser */
+
+// #############################################################################
+// INTEGRATION TEST
+describe('Aldryn Events tests: ', function () {
+    var aldrynTestsLibrary = require('../pages/alteli.js');
+    var aldrynEventsPage = require('../pages/page.add.delete.event.js');
```

Comment:
than a proper name might be ``spec.event.crud`` crud means: create, update, delete